### PR TITLE
Legion Immunes

### PR DESCRIPTION
**Pull request by SeniorScore**
## About The Pull Request

Adds funny gunslinger vex loadout and nightstalker vex helmet (from DR2). I wanted a corgi vex helmet yet here we are.

## Why It's Good For The Game

Dual Colt SAAs go BR-BR-BR

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog

:cl:
/:cl:

### CODE REVIEW

**Comment by SeniorScore** [comment]
ADDS IMMUNES, LEGION CAMP-DUTY ROLE, NOW TOO I GUESS. DEATH TO SLAVEMASTER

**Comment by Jawnner** [comment]
Gunslinger loadout is eh and doesn't really fit Legion like that. I'd replace the dual SAAs w/ a Marksman Carbine and a .357 revolver.

Replace the Mountain Bear pistol with a lever-action and buckshot shells, keep ripper

Replace Desert Fox 10mm w/ grease gun, .44 revolver

Give the Prime Dec the assault carbine and a scope underneath leads from the rear loadout

**Comment by SeniorScore** [comment]
Dumped the Vex loadout changes, nightstalker helmet is still in, unused is all.

Immunes should be ready to roll though.

File: code/__DEFINES/jobs.dm [hunk 1]
```diff
@@ -94,7 +94,7 @@
 #define F13RECRUITLEG	(1<<12)
 #define F13AUXILIA		(1<<13)
 #define F13LEGIONSLAVE	(1<<14)
-#define F13SLAVEMASTER	(1<<15)
+#define F13IMMUNE		(1<<15)
```
Comment (Looreydev) [inline code/__DEFINES/jobs.dm:97]:
```suggestion
#define F13IMMUNE	(1<<15)
```

File: code/modules/jobs/job_types/legion.dm [hunk 2]
```diff
@@ -898,48 +897,42 @@ commented out pending rework*/
...
+/datum/outfit/job/CaesarsLegion/Legionnaire/f13immune
+	name = "Immune"
+	jobtype = /datum/job/CaesarsLegion/Legionnaire/f13immune
+	id = /obj/item/card/id/dogtag/legimmune
 	mask = /obj/item/clothing/mask/bandana/legion/camp
 	uniform = /obj/item/clothing/under/f13/legskirt
-	suit = /obj/item/clothing/suit/armor/f13/legion/prime/slavemaster
-	head = /obj/item/clothing/head/helmet/f13/legion/prime/slavemaster
+	suit = null
+	head = null
```
Comment (Looreydev) [inline code/modules/jobs/job_types/legion.dm:920]:
```suggestion
```

File: code/modules/jobs/job_types/legion.dm [hunk 3]
```diff
@@ -898,48 +897,42 @@ commented out pending rework*/
...
 	glasses = /obj/item/clothing/glasses/sunglasses
-	shoes = /obj/item/clothing/shoes/roman
-	r_pocket = /obj/item/restraints/handcuffs
+	shoes = /obj/item/clothing/shoes/f13/military/leather
+	r_pocket = null
```
Comment (Looreydev) [inline code/modules/jobs/job_types/legion.dm:923]:
```suggestion
```

File: code/modules/jobs/job_types/legion.dm [hunk 4]
```diff
@@ -898,48 +897,42 @@ commented out pending rework*/
...
 		/obj/item/storage/bag/money/small/legenlisted = 1,
-		/obj/item/melee/onehanded/machete = 1,
-		/obj/item/ammo_box/shotgun/bean = 1,
-		/obj/item/melee/onehanded/slavewhip = 1,
-		/obj/item/razor = 1,
-		/obj/item/restraints/legcuffs/bola = 1,
-		/obj/item/electropack/shockcollar/explosive = 1,
-		/obj/item/warpaint_bowl
+		/obj/item/melee/onehanded/machete/forgedmachete = 1,
+		/obj/item/reagent_containers/pill/patch/healingpowder = 2,
```
Comment (Looreydev) [inline code/modules/jobs/job_types/legion.dm:929]:
```suggestion
		/obj/item/reagent_containers/pill/patch/healingpowder = 2
```

**Review by Looreydev — CHANGES_REQUESTED** [review]

File: code/__DEFINES/jobs.dm [hunk 5]
```diff
@@ -94,7 +94,7 @@
 #define F13RECRUITLEG	(1<<12)
 #define F13AUXILIA		(1<<13)
```
Comment (Looreydev) [inline code/__DEFINES/jobs.dm:95]:
```suggestion
#define F13AUXILIA	(1<<13)
```

**Review by Looreydev — CHANGES_REQUESTED** [review]

**Review by Looreydev — APPROVED** [review]

**Pull request merged**